todo_app/data/items.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+from datetime import date
+from enum import Enum
+from typing import Optional
+
+
+class Status(str, Enum):
+    NOT_STARTED = "Not Started"
+    IN_PROGRESS = "In Progress"
+    COMPLETED = "Completed"
+
+
+@dataclass
+class Item:
+    id_: str
```

Comment:
This feels like it's probably a Python pattern I wasn't aware of! Why `id_` over `id`?

Reply:
`id` is a python [builtin function](https://docs.python.org/3/library/functions.html#id), and I don't really like naming things the same as them